mlos_core/mlos_core/tests/optimizers/optimizer_test.py

```diff
@@ -298,3 +298,69 @@ def test_optimizer_type_defs(optimizer_class: Type[BaseOptimizer]) -> None:
     """
     optimizer_type_classes = {member.value for member in OptimizerType}
     assert optimizer_class in optimizer_type_classes
+
+
+@pytest.mark.parametrize(('optimizer_type', 'kwargs'), [
+    # Default optimizer
+    (None, {}),
+    # Enumerate all supported Optimizers
+    *[(member, {}) for member in OptimizerType],
+    # Optimizer with non-empty kwargs argument
+])
+def test_mixed_input_space_types(optimizer_type: Optional[OptimizerType], kwargs: Optional[dict]) -> None:
+    """
+    Toy problem to test the optimizers.
+    """
+    max_iterations = 10
+    if kwargs is None:
+        kwargs = {}
+
+    def objective(point: pd.DataFrame) -> pd.Series:
+        # mix of hyperparameters, optimal is to select the highest possible
+        ret: pd.Series = point["x"] + point["y"]
+        return ret
+
+    input_space = CS.ConfigurationSpace(seed=2169)
```

Comment:
is that particular seed number important?

Reply:
Elsewhere I think we have a `SEED` that you can reference.

Reply:
No reason, saw that previous tests were manually defining seeds. `mlos_core` does not have a SEED defined anywhere, but I see [`mlos_bench/tests/__init__.py:40`](https://github.com/microsoft/MLOS/blob/main/mlos_bench/mlos_bench/tests/__init__.py#L40-L42) does. I see this repo likes `SEED = 42` as the default.

For the sake of isolation, I'll add `SEED = 42` to `mlos_core/tests/__init__.py` rather than importing SEED from `mlos_bench` to be used to reference in tests.